github/repos_test.go

```diff
@@ -926,3 +926,43 @@ func TestPullRequestReviewsEnforcementRequest_MarshalJSON_nilDismissalRestirctio
 		t.Errorf("PullRequestReviewsEnforcementRequest.MarshalJSON returned %+v, want %+v", string(json), want)
 	}
 }
+
+func TestRepositoriesService_ListAllTopics(t *testing.T) {
+	setup()
+	defer teardown()
+
+	mux.HandleFunc("/repos/o/r/topics", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "GET")
+		fmt.Fprint(w, `{"names":["go", "go-github", "github"]}`)
+	})
+
+	got, _, err := client.Repositories.ListAllTopics(context.Background(), "o", "r")
+	if err != nil {
+		t.Errorf("Repositories.ReplaceAllTopics returned error: %v", err)
```

Comment:
I think you meant "Repositories. ListAllTopics" in the error message, not "Repositories.ReplaceAllTopics".

Also, use `t.Fatalf` instead of `t.Errorf` here. If `err != nil`, that's a fatal error, we don't want to continue the test, since comparing the result of `got` is meaningless.